pkg/api/norman/customization/monitor/prometheus_manager.go

```diff
@@ -126,7 +126,7 @@ func (q *Queries) Do(querys []*PrometheusQuery) (map[string][]*TimeSeries, error
 }
 
 func (q *Queries) GetLabelValues(labelName string) ([]string, error) {
-	value, _, err := q.api.LabelValues(q.ctx, labelName, []string{}, time.Now().Add(-time.Hour), time.Now())
+	value, _, err := q.api.LabelValues(q.ctx, labelName, nil, time.Now().Add(-time.Hour), time.Now())
```

Comment:
This change looks irrelevant to the scope of the PR to me.

Reply:
Ah yes. This is leftover from my original switch, and should not be changed. Will update.

Reply:
Fixed.